### PR TITLE
Create Unit tests covering remove fleet Management device(s) from an existing device group

### PR DESCRIPTION
**Pull request by Avielyo10**
# Description

Create Unit tests covering remove fleet Management device(s) from an existing device group

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [x] Tests update
- [ ] Refactor

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I run `go fmt ./...` to check that my code is properly formatted
- [x] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

File: pkg/services/devicegroups_test.go [hunk 1]
```diff
@@ -282,6 +287,70 @@ var _ = Describe("DeviceGroupsService basic functions", func() {
...
+				Expect(err).To(BeNil())
+
+				deletedDevices, delErr := deviceGroupsService.DeleteDeviceGroupDevices(account, deviceGroupID, []models.Device{savedDeviceGroup.Devices[3], fakeDevice})
+				Expect(delErr).NotTo(BeNil())
+				Expect(delErr.Error()).To(Equal(new(services.DeviceGroupDevicesNotFound).Error()))
+				Expect(deletedDevices).To(BeNil())
+			})
+
+			// delete device from another device-group
+			It("should not remove non existing devices from device-group", func() {
```
Comment (AaronH88) [inline pkg/services/devicegroups_test.go:323]: This description is the same as line 311, can this one of them be reworded so its easier to find if they fail?

**Review by AaronH88 — CHANGES_REQUESTED** [review]
One description change, then its good to merge

File: pkg/services/devicegroups_test.go (hunk 1 above)
Comment (Avielyo10) [reply pkg/services/devicegroups_test.go]: Done

File: pkg/services/devicegroups_test.go [hunk 2]
```diff
@@ -282,6 +287,70 @@ var _ = Describe("DeviceGroupsService basic functions", func() {
 				Expect(deletedDevicesIDS).To(BeEmpty())
 			})
 
+			It("should not delete non exsiting device from device-group", func() {
```
Comment (loadtheaccumulator) [inline pkg/services/devicegroups_test.go:290]: typo for existing

**Review by loadtheaccumulator — APPROVED** [review]
/lgtm

**Review by acosferreira — APPROVED** [review]
lgtm

**Review by AaronH88 — APPROVED** [review]

**Pull request merged**